src/Framework/LegacyPaymentGateways/Adapters/LegacyPaymentGatewayRegisterAdapter.php

```diff
@@ -67,30 +64,52 @@ static function (int $donationId, string $newStatus, string $oldStatus) use (
     /**
      * Adds new payment gateways to legacy list for settings
      *
+     * @unreleased update admin_label to include version compatibility
      * @since 2.25.0 add is_visible key to $gatewayData
      * @since 2.19.0
      */
     public function addNewPaymentGatewaysToLegacyListSettings(array $gatewaysData, array $newPaymentGateways): array
     {
         foreach ($newPaymentGateways as $gatewayClassName) {
-            /* @var PaymentGatewayInterface $paymentGateway */
+            /* @var PaymentGateway $paymentGateway */
             $paymentGateway = give($gatewayClassName);
 
             $gatewaysData[$paymentGateway::id()] = [
-                'admin_label' => $paymentGateway->getName(),
+                'admin_label' => $this->getAdminLabel($paymentGateway),
                 'checkout_label' => $paymentGateway->getPaymentMethodLabel(),
-                'is_visible' => $this->supportsLegacyForm($paymentGateway),
+                'is_visible' => $this->supportsV2Forms($paymentGateway),
             ];
         }
+        $gatewaysData['offline']['admin_label'] .= " (v2)";
+        $gatewaysData['manual']['admin_label'] .= " (v2)";
```

Comment:
I think I'd like to also add a notice somewhere on the gateway page explaining what exactly this means. I think it will likely confuse people otherwise.

Reply:
I added this message:

<img width="1029" alt="Screenshot 2023-06-29 at 3 14 51 PM" src="https://github.com/impress-org/givewp/assets/10138447/509e0e85-09b1-42d2-ad27-2d7c39b8df4f">